test/models/user_test.exs

```diff
@@ -0,0 +1,86 @@
+defmodule Yggdrasil.UserTest do
+  use Yggdrasil.ModelCase
+
+  alias Yggdrasil.User
+
+  @valid_attrs %{username: "tester", password: "password", password_confirmation: "password"}
```

Comment:
If you made all these fixture attributes functions then they could build on top of each other.

Example:
```elixir
def valid_user do
    %{ username: "tester", password: "password", password_confirmation: "password" }
end

def missing_username do
    valid_user |> Map.delete :username
end
```

Certainly doesn't save any code length, but maybe some on repetition. You could also move these out to a `Yggdrasil.Fixtures.User` module. That's a normal pattern for test data that I use. You use your own judgement though.